File: Devskill/c++/DCP-194: Number Formatting.cpp
#include<bits/stdc++.h>

using namespace std;
string f(char c){
    switch(c){
    case '8':
    return "Robi";
    break;
    case '7':
    return "Grameenphone";
    break;
    case '1':
    return "Citycell";
    break;
    case '5':
    return "Teletalk";
    break;
    case '6':
    return "Airtel";
    break;
    case '9':
    return "Banglalink";
    break;
    }
}
int main(void){
    int n;
    cin>>n;
    while(n--){
            string s;
            cin>>s;
            if(s[0]!='8'){
                cout<<"88"<<s<<" "<<f(s[2])<<endl;
            }
            else cout<<s<<" "<<f(s[4])<<endl;
        }
}

File: Devskill/c#/DCP-194: Number Formatting.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Test
{
	public static void Main()
	{	
		int n=int.Parse(Console.ReadLine());
	 	for(int i=1;i<=n;i++){
			string s = Console.ReadLine();
			Console.WriteLine(s.Length==11? "88"+s+" "+testme(s[2]): s+" "+testme(s[4]));
		}
	}
	public static string testme(char c){
		string s="";
		switch(c){
			case '1': s= "Citycell"; break;
			case '5': s= "Teletalk"; break;
			case '6': s= "Airtel"; break;
			case '7': s= "Grameenphone"; break;
			case '8': s= "Robi"; break;
			case '9': s= "Banglalink"; break;
		}
		return s;
	}
}
